tests/test_schemas.py

```diff
@@ -57,12 +62,53 @@ def test_parse_model(self):
         return
 
 class TestDataModelCsvParser:
-    def test_check_schema_definition(self):
-        return
-    def test_gather_csv_attributes_relationships(self):
-        return
-    def test_parse_csv_model(self ):
-        return
+    @pytest.mark.parametrize("data_model", ['example.model.csv'], ids=["csv"])
+    def test_check_schema_definition(self, helpers, data_model, dmcsvp:DataModelCSVParser):
+        """If the csv schema contains the required headers, then this function should not return anything. Check that this is so.
+        """
+        path_to_data_model = helpers.get_data_path(path=data_model)
+        model_df = load_df(path_to_data_model, data_model=True)
+        assert None == (dmcsvp.check_schema_definition(model_df = model_df))
+
+    @pytest.mark.parametrize("data_model", ['example.model.csv'], ids=["csv"])
+    def test_gather_csv_attributes_relationships(self, helpers, data_model, dmcsvp:DataModelCSVParser):
+        """The output of the function is a attributes relationship dictionary, check that it is formatted properly.
+        """
+        path_to_data_model = helpers.get_data_path(path=data_model)
+        model_df = load_df(path_to_data_model, data_model=True)
+
+        # Get output of the function:
+        attr_rel_dict = dmcsvp.gather_csv_attributes_relationships(model_df=model_df)
+
+        # Test the attr_rel_dict is formatted as expected:
+        # Get a key in the model
+        attribute_key = list(attr_rel_dict.keys())[0]
+
+        # Check that the structure of the model dictionary conforms to expectations.
+        assert True == (type(attr_rel_dict) == dict)
+        assert True == (attribute_key in attr_rel_dict.keys())
+        assert True == ('Relationships' in attr_rel_dict[attribute_key])
+        assert True == ('Attribute' in attr_rel_dict[attribute_key]['Relationships'])
+
```

Comment:
The `True ==` is redundant in these assert statements
`assert 'Attribute' in attr_rel_dict[attribute_key]['Relationships']`
would work for example